packages/react-components/react-rating-preview/src/components/RatingDisplay/useRatingDisplay.tsx

```diff
@@ -54,6 +54,8 @@ export const useRatingDisplay_unstable = (
       getIntrinsicElementProps('div', {
         ref,
         children: rootChildren,
+        role: 'img',
+        'aria-label': `${value} out of ${max} stars`,
```

Comment:
Also since the format of the `aria-label` is the same as what RatingItem would need, maybe the same prop for both? Potentially something like `valueText: (value: number) => string`?